app/main/views/frameworks.py

```diff
@@ -155,12 +187,12 @@ def framework_updates():
 def framework_updates_email_clarification_question():
 
     # Stripped input should not empty
-    clarification_question = escape(request.form.get(CLARIFICATION_QUESTION_NAME, '')).strip()
+    clarification_question = request.form.get(CLARIFICATION_QUESTION_NAME, '').strip()
 
     if not clarification_question:
-        return _framework_updates_page("Question cannot be empty")
+        return framework_updates("Question cannot be empty")
     elif len(clarification_question) > 5000:
-        return _framework_updates_page("Question cannot be longer than 5000 characters")
+        return framework_updates("Question cannot be longer than 5000 characters")
```

Comment:
When this error happens the submitted text is not re-populated into the text box.  That would be quite frustrating if I'd just written a genuine 5001+ character question.